src/disjoint_mut.rs

```diff
@@ -219,30 +224,32 @@ impl<T: AsMutPtr> DisjointMut<T> {
     /// during the lifetime of the returned borrow.
     ///
     /// [`index_mut`]: DisjointMut::index_mut
+    #[track_caller]
```

Comment:
track_caller should be cfg_attr here, it's not zero-cost.

Reply:
Oh, it's not? I thought it was just a debug info thing.  I can change that then.

Reply:
Would it still be useful here and in `DisjointMutIndex::get_mut` for the bounds check?

Reply:
I don't think so, this is fine as is.